cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -230,28 +231,31 @@ protected void scheduleWorkflow(Schedule schedule, String workflowName) {
   }
 
   /**
-   * Schedule the specified {@link Workflow}
+   * Schedules the specified {@link Workflow} using a time-based schedule.
    * @param scheduleName the name of the Schedule
    * @param cronTab the crontab entry for the Schedule
    * @param workflowName the name of the Workflow
    */
+  @Deprecated
```

Comment:
It is a good practice to add a @deprecated tag in the javadocs, too, with a suggestion of what to use instead. See addProcedure()... and this applies to other places in this PR, too.